### PR TITLE
Gesture for opening texteditor with last previously edited file

### PR DESCRIPTION
**Pull request by smartscripts-nl**
Several taps are required to re-open the last edited file in the text-editor. By adding a gesture for that, we can quicken that procedure.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6600)

### CODE REVIEW

File: frontend/dispatcher.lua [hunk 1]
```diff
@@ -48,6 +48,7 @@ local settingsList = {
     stats_calendar_view = { category="none", event="ShowCalendarView", title=_("Statistics calendar view"), device=true, separator=true,},
     history = { category="none", event="ShowHist", title=_("History"), device=true,},
     open_previous_document = { category="none", event="OpenLastDoc", title=_("Open previous document"), device=true,},
+    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = 'Texteditor: re-open last edited file', device = true, },
```
Comment (yparitcher) [inline frontend/dispatcher.lua:51]: please use `gettext` _() for the title so it can be translated

Comment (Frenzie) [reply frontend/dispatcher.lua:51]: I think the text can probably be shortened a little without losing any clarity.
```suggestion
    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = _("Text editor: open last file"), device = true, },
```

File: frontend/dispatcher.lua [hunk 2]
```diff
@@ -77,6 +77,7 @@ local settingsList = {
     calibre_search = { category="none", event="CalibreSearch", title=_("Search in calibre metadata"), device=true,},
     calibre_browse_tags = { category="none", event="CalibreBrowseTags", title=_("Browse all calibre tags"), device=true,},
     calibre_browse_series = { category="none", event="CalibreBrowseSeries", title=_("Browse all calibre series"), device=true, separator=true,},
+    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = 'Texteditor: open last file', device = true, separator = true, },
```
Comment (yparitcher) [inline frontend/dispatcher.lua:80]:
```suggestion
    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title =_("Texteditor: open last file"), device = true, separator = true, },
```

@smartscripts-nl please fix this in one of your other PRs

Comment (smartscripts-nl) [reply frontend/dispatcher.lua:80]: I've done that, in #6608

**Pull request merged**